spyder_kernels/customize/spyderpdb.py

```diff
@@ -13,6 +13,7 @@
 import sys
 import traceback
 from collections import namedtuple
+from typing import Tuple, Optional
```

Comment:
```suggestion
```

This package still supports Python 2, so we need to remove this and the typing annotations you added below. We will migrate to Python 3 only in Spyder-kernels 3.